server_addon/maya/client/ayon_maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -274,19 +275,18 @@ def get_sequence(self, filepath, pattern="%04d"):
             list: file sequence.
```

Comment:
Let's make sure our IDEs can also start warning us if we're returning the wrong types, etc.
```suggestion
            list[str]: file sequence.
```